src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -466,6 +468,11 @@ public static boolean downloadDistribution(PrintStream printStream, String distr
                         OSUtils.getUserAgent(getCurrentBallerinaVersion(), ToolUtil.getCurrentToolsVersion(),
                                 distributionType));
                 conn.setRequestProperty("Accept", "application/json");
+                if (testMode || TEST_MODE) {
+                    conn.setRequestProperty("testMode", "true");
+                } else {
+                    conn.setRequestProperty("testMode", String.valueOf(testMode));
```

Comment:
Do we really need this `else` ? if default value is `false`, then `testMode` variable will be `false` here which the default value.